### DISCUSSION TITLE
turbo dev with a Rollup-packaged package

### DISCUSSION THREAD

**Discussion by baptisteArno** [vercel/turborepo#387] [Help] [answered]
I have a Next.js app that has a rollup-built package as a  dependency.

I set the `dev` command of the package as `yarn rollup -c --watch`. When I save a file from the package, it restarts the Next.js app but modifications are not applied, I need to save again a file from the package to reload again and then I see the updates on the Next.js app.

Any idea why?

**Comment by anthonyshew** [comment, marked as answer]
This is because the sub-processes running the compilation are two separate tasks.

Follow along with #986 for discussion on this.
